### PR TITLE
Add new option to suppress output errors

### PR DESCRIPTION
**Pull request by theisensanders-wf**
Every time I saved a dart file that had a syntax error, the 'OverReact Format On Save' extension would pop up the console. I was getting tired of dismissing this, so I added an option to disable this behavior.

Additionally, VSCode is discouraging use of the `*` activationEvent so I changed it to be `onLanguage:dart`. This also may improve vscode startup time when using this extension, though I haven't measured this.

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
<details>
<summary>(5) Vulnerable direct dependencies were detected</summary>
<li>1 vulns in <code>ajv < 6.12.3</code> via <code>yarn.lock</code></li>
<li>1 vulns in <code>glob-parent < 5.1.2</code> via <code>yarn.lock</code></li>
<li>1 vulns in <code>lodash < 4.17.21</code> via <code>yarn.lock</code></li>
<li>1 vulns in <code>minimist < 1.2.6</code> via <code>yarn.lock</code></li>
<li>1 vulns in <code>y18n < 4.0.1</code> via <code>yarn.lock</code></li>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=vs-code-format-on-save">this Splunk dashboard</a> for more CVE details
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ RUN yum install -y \
 
 RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash && \
     . ~/.nvm/nvm.sh && \
-    nvm install node && \
+    nvm install 17 && \
```
Comment (theisensanders-wf) [inline Dockerfile:13]: Node 18 [breaks the build](https://ci.webfilings.com/build/3652459)

**Review by joebingham-wk — APPROVED** [review]
Thank you for the PR!!

+10

- [x] Code changes look good
- [x] QA'd by installing from the included .vsix and verified it changed the behavior as expected
- [x] Verified generally that formatting on save, enabling, and disabling all still function as expected

**Comment by joebingham-wk** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**